pyproject.toml

```diff
@@ -63,8 +63,8 @@ environments = [
 ]
 
 [tool.ruff]
-# Allow lines to be as long as 80.
-line-length = 80
+# Use widescreen monitors 😎😎😎
+line-length = 120
```

Comment:
Here is an example of readability getting worse at least when looking at Github diffs:

<img width="1059" height="572" alt="image" src="https://github.com/user-attachments/assets/e2e33fe3-4b4d-4928-926b-bb86eaf0919a" />

or here

<img width="951" height="158" alt="image" src="https://github.com/user-attachments/assets/3acfb764-b3c2-4c25-9e15-5e5c169620f1" />